ocean_lib/assets/v4/test/test_asset.py

```diff
@@ -105,6 +108,7 @@ def test_asset_utils(web3):
         nft=nft,
         datatokens=data_tokens,
         event=event,
+        purgatory=purgatory,
```

Comment:
no, assets that are in purgatory are never returned by Aquarius, so you don't need to include this in the asset.